src/environment.js

```diff
@@ -0,0 +1,42 @@
+import Web3 from 'web3'
+import {
+  getDefaultEthNode,
+  getEthNetworkType,
+  getIpfsGateway,
+} from './local-settings'
+import { getNetworkConfig } from './network-config'
+
+const networkType = getEthNetworkType()
+
+export const ipfsDefaultConf = {
+  gateway: getIpfsGateway(),
+}
+
+const networkConfig = getNetworkConfig(networkType)
+export const network = networkConfig.settings
```

Comment:
I think it could be nice to make the network dynamic from the start (maybe as a hook?).

Should we also start from the changes made in Brett’s PR? https://github.com/aragon/aragon/pull/1485